engine/src/main/resources/templates/ResourceOptimizationService/dailyEmailBody.html

```diff
@@ -0,0 +1,26 @@
+{#include ResourceOptimizationService/insightsEmailBody}
+{#content-title}Daily Resource Optimization Service Summary - {action.context.start_time.toStringFormat()}{/content-title}
+{#content-body}
+<tr>
+    <td class="rh-content__block">
+        <p>Resource Optimization Service has identified <b> {action.context.system_count}</b> system{#if action.context.system_count > 1}s{/if} with new suggestion{#if action.context.system_count > 1}s{/if}.
+        </p>
+    </td>
+</tr>
+<tr>
+    <td class="rh-content__block">
+        <table align="center">
+            <tr>
+                <td class="rh-cta-link" align="center">
+                    <a target="_blank" href="https://console.redhat.com/insights/ros">
```

Comment:
This will make the link work on both stage and prod (right now it only works on prod):
```suggestion
                    <a target="_blank" href="{environment.url}/insights/ros">
```

Reply:
Got it 👍🏻 thanks!